modules/installation-launching-installer.adoc

```diff
@@ -301,26 +293,25 @@ https://ovirtlab.example.com/ovirt-engine/api
 ----
 endif::openshift-origin[]
 +
-.. For `Is the oVirt CA trusted locally?`, enter `Yes` since you have already set up a CA certificate. Otherwise, enter `No`.
-.. For `oVirt's CA bundle`, if you entered `Yes` for the preceding question, copy the certificate content from `/etc/pki/ca-trust/source/anchors/ca.pem` and paste it here. Then, press `Enter` twice. Otherwise, if you entered `No` for the preceding question, this question does not appear.
-.. For `oVirt engine username`, enter the user name and profile of the {rh-virtualization} administrator using this format:
+.. The installer automatically generates a CA certificate. For `Would you like to use the above certificate to connect to Engine?`, enter `Yes`. Otherwise, enter `No`. If you answer `No`, you need to install {product-title} in insecure mode.
+.. For `Engine username`, enter the user name and profile of the {rh-virtualization} administrator using this format:
 +
 [source,terminal]
 ----
-<username>@<profile> <1>
+$ <username>@<profile> <1>
 ----
 +
 <1> For `<username>`, specify the user name of an {rh-virtualization} administrator. For `<profile>`, specify the login profile, which you can get by going to the {rh-virtualization} Administration Portal login page and reviewing the *Profile* dropdown list. Together, the user name and profile should look similar to this example:
 +
 [source,terminal]
 ----
-admin@internal
+$ admin@internal
```

Comment:
Same here about "$"

Also just a note that I don't think this is formatted correctly. It looks like it's a continuation of the above callout, but the example isn't indented with it. I know that callouts are a little funky when it comes to things like this. I would consider just adding the `admin@internal` example into the above callout's sentence.

Reply:
Fixed. Take a look when you can.